internal/nodes/avi_model_advl4_translator.go

```diff
@@ -471,6 +472,7 @@ func (o *AviObjectGraph) ConstructSharedVipSvcLBNode(sharedVipKey, namespace, ke
 			if infraSettingAnnotation, ok := svcObj.GetAnnotations()[lib.InfraSettingNameAnnotation]; ok && infraSettingAnnotation != "" {
 				serviceObject = svcObj.DeepCopy()
 			}
+			appProfile = svcObj.GetAnnotations()[lib.LBSvcAppProfileAnnotation]
```

Comment:
Can we change this to `if appProfile, ok:=svcObj.GetAnnotations()[lib.LBSvcAppProfileAnnotation]; ok`

Reply:
Done